lib/chef-acceptance/chef_runner.rb

```diff
@@ -13,12 +13,14 @@ class ChefRunner
     attr_reader :test_suite
     attr_reader :recipe
     attr_reader :duration
+    attr_reader :timeout
 
-    def initialize(test_suite, recipe)
+    def initialize(test_suite, recipe, timeout)
       @test_suite = test_suite
       @acceptance_cookbook = test_suite.acceptance_cookbook
       @recipe = recipe
       @duration = 0
+      @timeout = timeout
```

Comment:
Allowing timeout to be set here seems like an issue since we already set and default the value in Application.  Maybe run! takes options.  That way you can call `runner.run!(timeout: timeout)`